app/javascript/packages/document-capture/components/document-side-acuant-capture.tsx

```diff
@@ -52,13 +53,14 @@ function DocumentSideAcuantCapture({
   errors,
   onError,
   className,
+  isReviewStep = false,
```

Comment:
it looks like we're passing `isReviewStep` at every instantiation of this component, WDYT of making it required instead of optional like this?
```suggestion
  isReviewStep,
```

Reply:
Makes sense, will change.